### ISSUE TITLE
Ability to deactivate the smooth scrolling feature for individual apps [enhancement]

### ISSUE THREAD

**Issue by Lappun0** [noah-nuebling/Mac-Mouse-Fix#763]
## Description
A whitelist or blacklist for the smooth scrolling features.

## Rationale
Some apps (specifically games) expect the standard scrolling input from MacOS and sometimes even apply their own smoothing to it. The scrolling input from Mac Mouse Fix can cause all sorts of issues in these applications and the ability to disable it for them would solve this problem.

## Additional info
I'm currently using an App called Mos, which has exactly this kind of system and it allows me to always keep it open and to be able to use all sorts of apps. The ability to simulate a trackpad and the much smoother scrolling make me want to move to Mac Mouse Fix, but I can only do that if it has this Blacklist feature.

**Comment by TinyMito** [comment]
I agree, I am looking forward a feature per app profile. Currently, I am having issue with Fusion360 with Mac Mouse Fix.

https://github.com/noah-nuebling/mac-mouse-fix/issues/767

**Comment by lollilol** [comment]
duplicate of #598
